Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Storage/Queries/DriverFee.hs

```diff
@@ -187,6 +187,20 @@ findWindows (Id driverId) from to limitVal offsetVal =
     (Just limitVal)
     (Just offsetVal)
 
+findWindowsWithoutLimit :: MonadFlow m => Id Person -> UTCTime -> UTCTime -> m [DriverFee]
+findWindowsWithoutLimit (Id driverId) from to = do
+  findAllWithOptionsKV
+    [ Se.And
+        [ Se.Is BeamDF.startTime $ Se.GreaterThanOrEq from,
+          Se.Is BeamDF.endTime $ Se.LessThanOrEq to,
+          Se.Is BeamDF.driverId $ Se.Eq driverId,
+          Se.Is BeamDF.status $ Se.Not $ Se.Eq INACTIVE
```

Comment:
We should not show invoice for ongoing rides too, right?

Reply:
I added the condition for ONGOING in the query.